torch/__config__.py

```diff
@@ -20,3 +20,8 @@ def _cxx_flags():
 def parallel_info():
     r"""Returns detailed string with parallelization settings"""
     return torch._C._parallel_info()
+
+
+def get_cpu_capability():
+    r"""Returns cpu capability"""
```

Comment:
nit: it would be good to be clearer on what we return here and share the list of possible values since I guess the user will have to do string matching on them?

Reply:
Yes, sure, I'll update the docstring together with fixing merge conflict